src/jdk.jdi/share/man/jdb.1

```diff
@@ -230,15 +230,25 @@ Runs the application in the Java HotSpot VM server.
 .RE
 .TP
 .B \f[CB]\-J\f[R]\f[I]option\f[R]
-Passes \f[I]option\f[R] to the JVM, where option is one of the options
-described on the reference page for the Java application launcher.
+Passes \f[I]option\f[R] to the JDB JVM, where option is one of the
+options described on the reference page for the Java application
+launcher.
```

Comment:
FYI, the change here is replacing "JVM" with "JDB JVM". The rest is just formatting.